cliquet/tests/test_cache.py

```diff
@@ -156,12 +159,86 @@ def test_ping_returns_false_if_unavailable(self):
     def test_ping_logs_error_if_unavailable(self):
         pass
 
+    def test_cache_prefix_is_set(self):
+        settings_prefix = self.settings.copy()
+        settings_prefix['cache_prefix'] = 'prefix_'
+        config_prefix = self._get_config(settings=settings_prefix)
+
+        # initiating cache backend with prefix:
+        backend_prefix = self.backend.load_from_config(config_prefix)
```

Comment:
Why cannot you just do ``backend_prefix = self.backend.load_from_config(prefix="prefix_")`` here?